src/FubuMVC.Core/ServiceBus/Configuration/ChannelGraph.cs

```diff
@@ -112,6 +111,20 @@ public Uri ReplyChannelFor(string protocol)
             return _replyChannels[protocol];
         }
 
+        public List<string> GetAcceptedContentTypesForChannel(Uri channelUri)
+        {
+            var acceptedContentTypes = new List<string>();
+            var node = this.FirstOrDefault(x => x.Channel.Address == channelUri) ?? this.FirstOrDefault(x => x.Uri == channelUri);
+
+            acceptedContentTypes.AddRange(node?.AcceptedContentTypes ?? Enumerable.Empty<string>());
+            acceptedContentTypes.Add(node?.DefaultContentType);
+            acceptedContentTypes.Add(node?.DefaultSerializer?.ContentType);
+            acceptedContentTypes.AddRange(AcceptedContentTypes);
+            acceptedContentTypes.Add(DefaultContentType);
+
+            return acceptedContentTypes.Where(x => x != null).Distinct().ToList();
```

Comment:
Wasn't the plan.  My original intent was that the accepted content types would make it into the envelope's header but I noticed that wasn't happening either.  Going to fix that and add a test for it. I was trying to keep that header as small as possible if they configured the channels oddly and got duplicates.